nfd-worker-daemonset.yaml.template

```diff
@@ -199,4 +207,25 @@ data:
     #            vendor: ["15b3"]
     #            device: ["1014", "1017"]
     #          loadedKMod : ["vendor_kmod1", "vendor_kmod2"]
+    #    - name: "feature.by.nodename"
+    #      value: customValue
+    #      matchOn:
+    #        - nodename: ["worker-0", "my-.*-node"]
 ### <NFD-WORKER-CONF-END-DO-NOT-REMOVE>
+---
+## Example for more custom configs in an additional configmap (3/3)
+#apiVersion: v1
+#kind: ConfigMap
+#metadata:
+#  name: more-custom-configs
```

Comment:
nit: maybe it's my problem, but this is just somehow funny name 😄 . How about e.g. `custom-source-extra-config`, `custom-source-extra-rules` , `custom-extra-rule` or smth (you can probably come up with better name)?